EverythingSettings.xaml

```diff
@@ -66,5 +68,10 @@
                 </DataTemplate>
             </ComboBox.ItemTemplate>
         </ComboBox>
+
+        <TextBlock Name ="tbFastSortWarning"  Grid.Row="4" Grid.Column="2" Margin="10 5 10 0" TextAlignment="Left"
+                   Text="{DynamicResource flowlauncher_plugin_everything_nonfastsort_warning}"
```

Comment:
We may also need to notify user when everything is not running (which we cannot check whether the option is fast sort except sorting by name) .

Reply:
done